server/src/index.js

```diff
@@ -1,27 +1,34 @@
-require('dotenv').config()
+require('dotenv').config();
 import http from 'http';
 import https from 'https';
 import Koa from 'koa';
 import Io from 'socket.io';
 import KoaBody from 'koa-body';
 import cors from 'kcors';
 import Router from 'koa-router';
-import bluebird from 'bluebird';
-import Redis from 'redis';
-import socketRedis from 'socket.io-redis';
 import Socket from './socket';
-import crypto from 'crypto'
+import crypto from 'crypto';
 import mailer from './utils/mailer';
 import koaStatic from 'koa-static';
 import koaSend from 'koa-send';
-import {pollForInactiveRooms} from './inactive_rooms';
-
-bluebird.promisifyAll(Redis.RedisClient.prototype);
-bluebird.promisifyAll(Redis.Multi.prototype);
-
-const redis = Redis.createClient(process.env.REDIS_URL)
+import { pollForInactiveRooms } from './inactive_rooms';
+import RedisStore from './store/Redis';
+import MemoryStore from './store/Memory';
+
+const storeBackend = process.env.STORE_BACKEND || 'redis';
```

Comment:
Just to keep this initialization file tidy, would you mind moving this, the switch/case and `getStore` into a separate module? Then we can just have `const store = getStore()` here